projects/vdk-plugins/vdk-jupyter/build-job-tutorial-temp/tutorial.md

```diff
@@ -0,0 +1,238 @@
+# Learn How to Develop a Data Job through the Jupyter UI
+Time Commitment: About 25-30 minutes.
+
+## Prerequisites
+
+Before you start with the development, please ensure you have the following:
+
+<details>
+  <summary><b>Installed Software and Services</b></summary>
+
+- **VDK:** Ensure that the Versatile Data Kit is installed and properly configured.
+- **[Control Service](https://github.com/vmware/versatile-data-kit/wiki/Interfaces#control-service:~:text=Parameterized%20SQL-,Control%20Service,-Job%20Lifecycle%20API):** Essential for orchestrating the execution of Data Jobs.
+- **Jupyter instance:** Needed to access Jupyter Notebooks.
+- **[vdk-jupyterlab-extension](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/README.md):** This extension integrates VDK with JupyterLab.
+- **[vdk-notebook](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-notebook/README.md):** Allows the execution of VDK jobs with notebooks.
+- **[vdk-ipython](https://github.com/vmware/versatile-data-kit/blob/main/projects/vdk-plugins/vdk-ipython/README.md):** Loads VDK functionalities into IPython environments, allowing enhanced interaction.
+
+</details>
+
+If you don’t know how to create jobs, please check out the [Creation Guide](create_tutorial.md).
```

Comment:
That should go in the details . One pre-requisite is a created data job . 

But it's not really required

Reply:
The details are for the installations i would rather keep this here as it is just one sentence